Packs/CommonScripts/ReleaseNotes/1_6_13.md

```diff
@@ -0,0 +1,6 @@
+
+#### Scripts
+##### ExtractDomainAndFQDNFromUrlAndEmail
+- Enhanced the script to check only tld, the script no longer supports urls and emails.
+- Enhanced the script to avoid extraction of zip files.
+- Enhanced the script to extract only full domain.
```

Comment:
```suggestion
- Enhanced the script to check only TLP, the script no longer supports URLs and emails.
- Enhanced the script to avoid extraction of ZIP files.
- Enhanced the script to extract only full domain.
```